lib/matplotlib/axes/_axes.py

```diff
@@ -4696,7 +4697,7 @@ def invalid_shape_exception(csize, xsize):
                       label_namer="y")
     @_docstring.interpd
     def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
-                vmin=None, vmax=None, alpha=None, linewidths=None, *,
+                vmin=None, vmax=None, colorizer=None, alpha=None, linewidths=None, *,
```

Comment:
This has to go after the `*` or we break API (even though passing 10 positional arguments is a Bad Idea ™️ if someone was doing that we would now get the value of `alpha` as `colorizer`).

Reply:
Side-remark: We should consider tightening on the kw-only settings. It helps the understanding to collect related parameters next to each other. I'd be willing to force people with Bad Ideas™️ to adjust in this case.

But this doesn't help for now because we'd need to go through the deprecation cycle. So @tacaswell's comment is valid anyway.

Reply:
Thank you for the feedback, I was unsure if it was OK to break the list of args for people with Bad Ideas™️, but I guess we should cater to everyone :)